Marlin/Marlin_main.cpp

```diff
@@ -5590,7 +5590,7 @@ void home_all_axes() { gcode_G28(true); }
                       r = delta_calibration_radius * 0.1;
           z_at_pt[CEN] +=
             #if HAS_BED_PROBE
-              probe_pt(cos(a) * r + dx, sin(a) * r + dy, stow_after_each, 1)
+              probe_pt(cos(a) * r + dx, sin(a) * r + dy, stow_after_each, 1, false)
```

Comment:
printable in G33 should always be false 

center probe probe_pt(dx, dy, stow_after_each, 1, false) no longer behaves weird; so that one is solved.

Reply:
Is there any reason you can think of with this flag set to false why we wouldn't do the `short_move` probe in `run_z_probe`? That changes was introduced by my spirit animal @tcm0116 in https://github.com/MarlinFirmware/Marlin/commit/f54e0fc90f5bc12d665c5d877b73c95bf76e582f#diff-1cb08de130a6ece2d1b5b9c37bcfef48R2384.

Reply:
The flag just skips the test that the probe as well should stay within the printable area; it should do nothing else.